src/regions/Area.js

```diff
@@ -41,7 +42,7 @@ const Area = types.union(
       )
         return ClassificationArea;
       // may be a tag itself or just its name
-      const objectName = sn.object.name || sn.object;
+      const objectName = Tree.cleanUpId(sn.object.name || sn.object);
```

Comment:
I'm not sure, but it may be that this change we also would like to wrap into a feature flag.